src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipsePage.xaml

```diff
@@ -1,14 +1,43 @@
 ﻿<Page
-    x:Class="SamplesApp.Windows_UI_Xaml_Shapes.EllipsePage"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+	x:Class="SamplesApp.Windows_UI_Xaml_Shapes.EllipsePage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
 	<StackPanel Spacing="5">
-		<TextBlock Text="This is a ellipse" />
-		<Ellipse x:Name="DEllipsePage"  Stroke="Black" Height="200" Width="250" StrokeThickness="5"/>
+		<TextBlock FontSize="22">This is an ellipse</TextBlock>
+		<Ellipse x:Name="DEllipsePage" Stroke="Black" Height="125" Width="150" StrokeThickness="5" HorizontalAlignment="Left"/>
```

Comment:
Might be good to have measure tests similar to the Image PR

Reply:
I tried that... the problem is `<Ellipse>` are rendered using native primitives and it's difficult to measure them afterwards.

For image, since the logic is very complex, it has been centralized.  For ellipse, it's just a min/max logic.

Reply:
The real solution would be to fully implement the measure+arrange phases as other controls, even for native elements.  It's not the case right now on iOS and Android, so some bugs are still remaining about the clipping.

It's not a regression, because it wasn't working before.  More time should be spend in the future for that.